student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,5 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_EQUAL_STRING(malloc_username_from_conf_file(), my_username() );
```

Comment:
It's a goof practice to use TEST_ASSERT_EQUAL_STRING_MESSAGE and add a message on failure.

Reply:
I have used TEST_ASSERT_EQUAL_STRING_MESSAGE. I initially used TEST_ASSERT_EQUAL_STRING but later changed it to  TEST_ASSERT_EQUAL_STRING_MESSAGE before the deadline.